src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -126,6 +126,10 @@ class AktivitetskravService(
         aktivitetskravRepository.getAktivitetskrav(personIdent = personIdent, connection = connection)
             .map { it.toAktivitetskrav() }
 
+    fun getAktivitetskravForPersons(personidenter: List<PersonIdent>): Map<PersonIdent, Aktivitetskrav> =
+        aktivitetskravRepository.getAktivitetskravForPersons(personidenter = personidenter)
```

Comment:
Hva skjer hvis det finnes flere `Aktivitetskrav` for en person? (kan være flere hvis det er startet ny vurdering eller personen har blitt generert som kandidat på nytt.). Da må vi sørge for at det er den nyeste som havner i mappet. Finnes det en test på det?

Reply:
Her er du god! Kan skrive en test på det og få det til å fungere 🙏

Reply:
Skrev en test på det, og la til `mapMostRecentAktivitetskrav(…)` for å sikre at vi henter det nyligst lagte aktivitetskravet. Si ifra om du har forslag til endringer 😊